### PR TITLE
Add jsxAttributeCompletionStyle setting

### PR DESCRIPTION
**Pull request by armanio123**
Typescript setting update: https://github.com/microsoft/TypeScript/pull/45903

### CODE REVIEW

File: extensions/typescript-language-features/package.nls.json [hunk 1]
```diff
@@ -124,6 +124,10 @@
 	"typescript.preferences.importModuleSpecifierEnding.minimal": "Shorten `./component/index.js` to `./component`.",
 	"typescript.preferences.importModuleSpecifierEnding.index": "Shorten `./component/index.js` to `./component/index`.",
 	"typescript.preferences.importModuleSpecifierEnding.js": "Do not shorten path endings; include the `.js` extension.",
+	"typescript.preferences.jsxAttributeCompletionStyle": "Preferred style for JSX attributes completions.",
```
Comment (andrewbranch) [inline extensions/typescript-language-features/package.nls.json:127]:
```suggestion
	"typescript.preferences.jsxAttributeCompletionStyle": "Preferred style for JSX attribute completions.",
```

File: extensions/typescript-language-features/package.nls.json [hunk 2]
```diff
@@ -124,6 +124,10 @@
...
+	"typescript.preferences.jsxAttributeCompletionStyle.auto": "Automatically select the completion style based on the attribute type.",
+	"typescript.preferences.jsxAttributeCompletionStyle.braces": "Enclosed in braces all of the JSX attribute values.",
+	"typescript.preferences.jsxAttributeCompletionStyle.none": "Don't apply any completions on JSX attribute values.",
```
Comment (andrewbranch) [inline extensions/typescript-language-features/package.nls.json:130]: What do you think about
```suggestion
	"typescript.preferences.jsxAttributeCompletionStyle.auto": "Insert `={}` or `=""` after attribute names based on the prop type.",	
	"typescript.preferences.jsxAttributeCompletionStyle.braces": "Insert `={}` after attribute names.",
	"typescript.preferences.jsxAttributeCompletionStyle.none": "Only insert attribute names.",
```

(The backticks will get interpreted and markdown and show up formatted as code.)

**Review by andrewbranch — CHANGES_REQUESTED** [review]

File: extensions/typescript-language-features/package.nls.json (hunk 2 above)
Comment (armanio123) [reply extensions/typescript-language-features/package.nls.json:130]: Sounds good. I was actually struggling to get to a good description.

**Comment by mjbvz** [comment]
Thanks! Merging this so users can start testing it out in the next VS Code release if they are using TS next

**Pull request merged**